src/sentry/static/sentry/app/components/group/sidebar.jsx

```diff
@@ -241,19 +259,23 @@ const GroupSidebar = createReactClass({
         <h6>
           <span>{t('Tags')}</span>
         </h6>
-        {group.tags.map(data => {
-          return (
+        {this.state.tagsWithTopValues && _.filter(group.tags.map(tag => {
```

Comment:
Is `group.tags` an array? If so, we should use native `Array.filter()`

Reply:
yeah this is filtering the result of the `map` to remove instances where `return tagWithTopValues && ...` below is undefined. 

I don't think there's really any case where we would have a tag from `group.tags`, but not the matching top values in `tagsWithTopValues` unless the API fails in some weird way. so this might not be necessary at all, but just making sure.